docs/prover/cli/options.md

```diff
@@ -2322,6 +2322,43 @@ _Configuration file_
 "override_base_config": "confs/base_settings.conf"
 ```
 
+(--url_visibility)=
+## `url_visibility`
+
+**Option values**
+```sh
+private|public
+```
+
+**What does it do?**
+This attribute controls the visibility of the report link generated by the CLI.
+To minimize the risk of unintentionally sharing a public URL, the CLI excludes anonymous access keys from the report link by default and returns only the private version.
+Enum values:
+    - private (default): Returns a report link without an anonymous access key.
+    - public: Returns a report link that includes an anonymous access key for public access.
+
+Note: In CI, the default behavior differs. The CLI will always output the public link to facilitate easy sharing among users.
```

Comment:
```suggestion
```{Note} In CI, the default behavior differs. The CLI will always output the public link to facilitate easy sharing among users.
```
```